src/start.clj

```diff
@@ -1,19 +1,33 @@
 (ns start
   (:gen-class)
```

Comment:
Looks like you inherited this namespace from me, sorry.
Should probably change the name, and probably don't need :gen-class.
I tested that the jar runs without AoT compilation.
Probably works without :gen-class too.

Reply:
actually I thought `start` was suitable for this when I started so it's my idea :P what name do you think it would be better? `daemon` or `main`?

Reply:
naming is hard...

Reply:
Yep.  And you're right. I was misremembering `zero.starter`.